packages/entity-server/.env.example

```diff
@@ -1,8 +1 @@
-PORT=
-
 API_CLIENT_SALT=
```

Comment:
I don't this we actually need `API_CLIENT_SALT` here. Must've been left in here by accident when we changed how initialising api-clients works. So safe to remove this whole file :+1: